### PR TITLE
Strategic vs Technical Skill Change

### PR DESCRIPTION
**Pull request by leenak17**
Title capitalization changed to uppercase to remain consistent.
Format in body change to skill type, example, explanation (consistent).
I would create a separate paragraph starting from the second "Smash Training" that explains strategic skills.
prerequisite --> necessary (concise)
for being --> become (your app is designed to be transformative)
Last sentence is restructured to be less wordy, and be more consequential (IF you use this app, THEN you can be a better player.)

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -166,18 +166,17 @@
     <v-row>
       <v-col cols="12">
         <v-card>
-          <v-card-title>Strategic vs. technical skill</v-card-title>
+          <v-card-title>Strategic vs. Technical Skill</v-card-title>
           <v-card-text>
             <p>
               <b>Smash Training</b> can teach you technical skills like spacing
-              aerial attacks, but it can't teach you strategic skills like
+              aerial attacks, because they improve with practice. <b>Smash Training</b> can't teach you strategic skills like
```
Comment (arxanas) [inline src/views/Home.vue:173]: > because they improve with practice

:+1:

File: src/views/Home.vue [hunk 2]
```diff
@@ -166,18 +166,17 @@
     <v-row>
       <v-col cols="12">
         <v-card>
-          <v-card-title>Strategic vs. technical skill</v-card-title>
+          <v-card-title>Strategic vs. Technical Skill</v-card-title>
```
Comment (arxanas) [inline src/views/Home.vue:169]: What is the rationale for changing everything to title-case? I was consistent throughout the site before, I think.

Comment (leenak17) [reply src/views/Home.vue:169]: I think after two years of being tested on journalistic standards I've become accustomed to title case. I'm biased but I think it improves readability and makes the title stand out even more than just bolding, but without being distracting. Provides key words and such. After looking at this website though I see that it isn't the standard everywhere so if you're consistent I suppose it doesn't matter.

Comment (arxanas) [reply src/views/Home.vue:169]: Undo this header change.

**Pull request merged**